<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medico_agregar_alergia_operacion_enfermedad</class>
 <widget class="QDialog" name="medico_agregar_alergia_operacion_enfermedad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_agregar_enfermedad">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>138</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelar_enfermedad">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background:none;</string>
     </property>
     <property name="text">
      <string>Enfermedad:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_enfermedad">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#818181;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background:none;</string>
     </property>
     <property name="text">
      <string>Descripción:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_descripcion_enfermedad">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#818181;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="btn_agregar_operacion">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelar_operacion">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_operacion">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#818181;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background:none;</string>
     </property>
     <property name="text">
      <string>Operación:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background:none;</string>
     </property>
     <property name="text">
      <string>Fecha:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDateEdit" name="fecha_opercacion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#818181;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_agregar_alergia">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelar_alergia">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
background:none;</string>
     </property>
     <property name="text">
      <string>Alergia:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_alergia">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#818181</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
